apps/astarte_appengine_api/lib/astarte_appengine_api/device/devices.ex

```diff
@@ -0,0 +1,47 @@
+defmodule Astarte.AppEngine.API.Device.Devices do
```

Comment:
Schema module names are usually singular: see the [example from Ecto](https://hexdocs.pm/ecto/Ecto.Schema.html#module-example)

```suggestion
defmodule Astarte.AppEngine.API.Device.Device do
```